docker-compose.debug.yml

```diff
@@ -150,27 +155,29 @@ services:
     working_dir: /usr/src/app
     volumes:
       - ./src/mail:/usr/src/app
-      - ${MAIL_APPLICATION_LOG_PATH:-./src/mail/log/application}:/usr/src/app/log/application
-      - ${MAIL_ACCESS_LOG_PATH:-./src/mail/log/access}:/usr/src/app/log/access
     command: "npm run dev"
     ports:
       - ${MAIL_PORT:-59705}:3000
     restart: unless-stopped
     healthcheck:
-      test: [ "CMD-SHELL", "wget -q -S -O - http://localhost:3000 2>&1 | grep 'HTTP/1.1 200 OK'" ]
+      test:
+        [
+          "CMD-SHELL",
+          "wget -q -S -O - http://localhost:3000 2>&1 | grep 'HTTP/1.1 200 OK'"
+        ]
       interval: 10s
       timeout: 10s
       retries: 5
 
   frontend:
-    image: node:14.15.5
+    image: node:16.19.0
```

Comment:
You can bump Node to `v18`, just use `NODE_OPTIONS=--openssl-legacy-provider` when building to prevent node from using `OpenSSL v3` which is causing issues.